CloudNotes/CloudNotes/View/MemoDetailView/MemoDetailView.swift

```diff
@@ -0,0 +1,34 @@
+//
+//  MemoDetailView.swift
+//  CloudNotes
+//
+//  Created by Luyan on 2021/09/02.
+//
+
+import UIKit
+
+class MemoDetailView: RootView {
+    lazy var detailTextView: UITextView = {
+        let textView = UITextView()
+        textView.backgroundColor = UIColor.systemGray4
+        textView.font = UIFont.preferredFont(forTextStyle: .body)
```

Comment:
```suggestion
        textView.font = .preferredFont(forTextStyle: .body)
```

개인 스타일이지만 이렇게 쓰는거랑 안쓰는거랑 어떤 차이가 있을까요?

Reply:
13, 14 라인 모두 
`textView.backgroundColor = UIColor. ~~`
`textView.font = UIFont. ~~ `
할당 연산 왼쪽에 있는 요소들이 모두 해당 타입을 유추할 수 있게 해주고 있는데, 각각 Color, font 가 중복되어 나타나고 있다고 생각되네요. 위와 같이 타입이 충분히 유추가 가능할 땐 쓰지 않는게 중복을 줄일 수 있을 것 같아요